### ISSUE TITLE
Add Country Outlines

### ISSUE THREAD

**Issue by TheMowgliMan** [TheMowgliMan/Country-Clash#28]
^^^

Should be done after #25.

**Comment by 3quarterscolon** [comment]
why `wontfix`?

**Comment by TheMowgliMan** [comment]
This was for proper listing of how complete the milestone is, more things needed to be done before we could work on this.
